resource-validator/src/test/scala/com/broadinstitute/dsp/resourceValidator/DbReaderGetKubernetesClustersToDeleteSpec.scala

```diff
@@ -198,8 +198,7 @@ final class DbReaderGetKubernetesClustersToDeleteSpec extends AnyFlatSpec with C
     }
   }
 
-  // The scenario below is to mimic a cluster with batch-pre-created nodepools, which we don't want to auto-delete
-  it should "NOT detect for removal: Kubernetes cluster with nodepools but no apps" taggedAs DbTest in {
+  it should "detect for removal: Kubernetes cluster with nodepools but no apps" taggedAs DbTest in {
```

Comment:
My expectations:
1. The **older** version of this test should fail **with** your change in `kubernetesClustersToDeleteQuery`.
2. The **newer** version of this test should fail **without** your change in `kubernetesClustersToDeleteQuery`.

Have you observed these?

Reply:
Yes, I played around with switching it back and forth